### PR TITLE
Windows agent silent mode

### PR DESCRIPTION
**Pull request by pedrovelho**
Add documentation of silent mode to run agent installation remotely.

### CODE REVIEW

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 1]
```diff
@@ -470,6 +470,73 @@ The main troubles you may have to face are the following ones:
...
+command prompt with full admin priviledges.
+
+The silent option works to execute the installer directly on the command line
+without the need for a graphical interfaces. When the silent option is activated
+we need to provide at least user and password as parameters, domain name is optional
+since it is automatically detected.
+
+
+For instance to install the latest windows agent go
+link:http://jenkins.activeeon.com/job/agent-windows/lastSuccessfulBuild/artifact/[here]
```
Comment (paraita) [inline src/docs/admin/ProActiveAdminGuide.adoc]: I don't think our jenkins platform is to be used publicly. Instead, we should give the link to either the respective git repo (for the community edition) or the breadcrumb navigation link through the client's account page on the activeeon website (where we usually provide those builds). What do you think @fviale  ?

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -470,6 +470,72 @@ The main troubles you may have to face are the following ones:
 * You get an *authentication error*: this is probably due to your default credentials file which cannot be found. In the "Connection" tab of the Configuration Editor (Figure 5.4, “Configuration Editor window - Connection Tab (Resource Manager Registration)”), you can choose the credentials file you want. You can select, for instance, the credentials file located at PROACTIVE_HOME/config/authentication/scheduler.cred or your own credentials file.
 * The node seems to be well started but you cannot see it in the Resource Manager interface : in this case, make sure that the *port number* is the good one. Do not forget that the runtime port number is incremented from the initial ProActive Resource Manager port number. You can see exactly on which port your runtime has been started looking at the log file described above.
 
+==== Install Agent Windows Using Silent Mode
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: Automate Windows Agent Installation

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -470,6 +470,72 @@ The main troubles you may have to face are the following ones:
...
+
+When installing the agent in batch on several desktop machines one might use the
+command line installer instead of the graphical one. This eases to deploy several
+installations in parallel. To do so it is requiered to have remote access to
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: Replace by: The automated installation can be run through a command line (cmd.exe) with administrator priviledge.

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 4]
```diff
@@ -470,6 +470,72 @@ The main troubles you may have to face are the following ones:
...
+command prompt with full admin priviledges.
+
+The silent option works to execute the installer directly on the command line
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: In order to launch the automated installation, the /S option must be passed in the command line to the ProActive Windows Agent installer. Several options are required, such as the user and password.

Here is an example of automated installation command:
....

Here is the full list of command line options which can be passed to the installer:

<make  a table with all the options>

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 5]
```diff
@@ -470,6 +470,60 @@ The main troubles you may have to face are the following ones:
...
+[cols=2*]
+|===
+
+|`/S` | Run silently without graphical interface, uninstall any previous installation
+
+|`/UN` | Run the uninstall only
+
+|`/ALLOW` | Allow all users to control the service
+
+|`/USER=user` | Associate `user` account to manage proactive agent service, creates user if does not exist
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: if it does not exist

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 6]
```diff
@@ -470,6 +470,60 @@ The main troubles you may have to face are the following ones:
...
+
+|`/PASSWORD=pass` | Define password for the proactive agent `user`
+
+|`/DOMAIN=dom` | Specify `dom` as the windows domain
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: . If this option is not used, the domain will be set to the machine host name.

**Pull request merged**